compose.yml

```diff
@@ -1,14 +1,16 @@
 version: '3' 
 
 services:
+
   bitcoind:
     image: kylemanna/bitcoind:latest
+    volumes:
+      - data:/bitcoin/.bitcoin
     ports: 
-      - "8332:8332"                           # mainnet
-      # - "18332:18332"                       # testnet
+      - "8332:8332"                                        # mainnet
+      - "18332:18332"                                      # testnet
     command: 
-      - "-chain=main"                         # mainnet
-      # - "-chain=test"                       # testnet
+      - "-chain=${COUNTERPARTY_NETWORK:-main}"             # export COUNTERPARTY_NETWORK=test for testnet
```

Comment:
can we have `COUNTERPARTY_NETWORK` be either `testnet` or `mainnet` and then do `${COUNTERPARTY_DOCKER_DATA:0:4}` for bitcoind CLI compatibility?